ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,40 @@
-﻿namespace ConsoleApp1
+﻿using System.Collections;
+using static System.Net.Mime.MediaTypeNames;
+using System.Net.Http;
+using Newtonsoft.Json;
+using System.Security.Principal;
+
+namespace ConsoleApp1
 {
 	internal class Program
 	{
-		static void Main(string[] args)
+        static HttpClient client = new HttpClient();
+
+        static void Main(string[] args)
 		{
-		}
-	}
+            var result = GetShiba();
+
+            result.Wait();
+
+            string[] result2 = result.Result
+                .Split(new[] { '[', ',', ']', '"', '\\' }, StringSplitOptions.RemoveEmptyEntries);
+            Load(result2);
+
+            Console.WriteLine();
+        }
+
+        static async Task<string> GetShiba(int count = 1)
+        {
+            HttpResponseMessage response = await client.GetAsync
```

Comment:
you can also move this logic to try-catch block and read String from client 
```
var response = await client.GetStringAsync
                ($"http://shibe.online/api/shibes?count={count}&urls=true&httpsUrls=false");
```

Reply:
okay, changed that